src/pages/config/ConfigDetailsPage.tsx

```diff
@@ -117,7 +117,10 @@ export function ConfigDetailsPage() {
                   <div className="flex flex-row space-x-2">
                     <div className="flex flex-col w-full object-contain">
                       <div className="flex flex-col mb-6 w-full">
-                        <div className="flex relative py-6 px-4 border-gray-300 bg-white rounded shadow-md flex-1 overflow-x-auto">
+                        <div
```

Comment:
I did tried implementing in a similar way you are suggesting in side panel, but it won't work when we have scenario where we have to show vertical scroll in case of content overflow.